sdk/containerservice/arm-containerservicefleet/review/arm-containerservicefleet.api.md

```diff
@@ -22,8 +22,6 @@ export interface AgentProfile {
 // @public
 export interface APIServerAccessProfile {
     enablePrivateCluster?: boolean;
-    enableVnetIntegration?: boolean;
-    subnetId?: string;
```

Comment:
are they newly added in preview version? I don't see the changelog for them in preview version either.